crates/sui-protocol-config/src/lib.rs

```diff
@@ -1222,7 +1222,7 @@ impl ProtocolConfig {
             max_arguments: Some(512),
             max_type_arguments: Some(16),
             max_type_argument_depth: Some(16),
-            max_pure_argument_size: Some(16 * 1024),
+            max_pure_argument_size: Some(1024 * 1024), // TODO change back to 16
```

Comment:
```suggestion
            max_pure_argument_size: Some(16 * 1024),
```
Should we?

Reply:
we need this bc otherwise we can't submit the state proofs

Reply:
Yeah I’m just saying that you had a TODO there, so handle it and delete it